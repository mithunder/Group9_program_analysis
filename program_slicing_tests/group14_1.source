module group141 :
 a := 1 ;
 b := 2 ;
 f := 3 ;
 c := 10 ;
 skip ;
 write a + b ;
 if true -> b := 4 ; b := 5
 fi;
 do false -> c := 89
  [] a < b -> b := 0 ; f := 2
  [] a > b -> skip ; abort 
 od ;
 if false ->
@programslicing="true"
   c := f ; g := h + b + a fi ;
 skip ;
 do true -> a := 2 ; write 2 + 4 od ;
 a := b
end
