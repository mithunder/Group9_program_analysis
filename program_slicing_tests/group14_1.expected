module group141 :
a := 1;
f := 3;
b := 5;
do
   (a<b) ->
      b := 0;
      f := 2
od;
if
   false ->
      c := f
fi
end
